src/GravityFormsExtensions.php

```diff
@@ -80,6 +80,11 @@ class GravityFormsExtensions
             'name' => 'native',
             'default_value' => 1,
         ],
+        [
+            'label' => 'Bluesky',
+            'name' => 'bluesky',
+            'default_value' => 0,
```

Comment:
As like at `Planet 4 > Social > Social sharing options`, I kept it not enabled by default